utils/data_loader_multifiles.py

```diff
@@ -160,6 +161,14 @@ def _open_file(self, year_idx):
       self.orography_field = _orog_file['orog']
     if self.precip:
       self.precip_files[year_idx] = h5py.File(self.precip_paths[year_idx], 'r')['tp']
+
+
+  @property
+  def data_array(self):
```

Comment:
This function basically loads the data from the first file in the list `self.files_paths`? I guess this is used somewhere else, maybe in your loader or something.

Reply:
Yes, this is used to replace this code: https://github.com/ai2cm/FourCastNet/blob/888cd74707af80a220be29d428959d1fe0860ccd/inference/inference.py#L149-L158